conductor/blueprints/authorization/controllers.py

```diff
@@ -69,21 +72,9 @@ def __call__(self):
         return PUBLIC_KEY
 
 
-class Sample:
```

Comment:
Initially it was written py3-only - https://github.com/openspending/os-conductor#overview (because it's app, not lib). So if py2 support is needed `(object)` for classes and
```
# -*- coding: utf-8 -*-
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```
should be added.

PS.
@pwalsh I don't know why `__call__` is used here but why do you consider it as an evil?